src/components/NavBar/NavbarDesktop.tsx

```diff
@@ -65,6 +66,9 @@ const NavBar: React.FC = () => (
       <Link to="/contact" title="Contact" text="Contact">
         <Contact />
       </Link>
+      <Link to="/blog" title="Blog" text="Blog">
+        <Blog />
+      </Link>
```

Comment:
Can we also add this link in `src/components/NavBar/Sidebar.tsx`? It is currently missing in the sidebar menu on mobile, so there would be no point of entry for mobile users. 

Notes: 
- "Contact" is also missing in this sidebar
- Also, we might want to include the "Blog" in the footer, under the "Explore" subheading.

Reply:
resolved